### DISCUSSION TITLE
Folders: hide ancestor tags in nested tag names

### DISCUSSION THREAD

**Discussion by kslstn** [standardnotes/forum#2555] [Suggestions] [answered]
**Is your feature request related to a problem? Please describe.**
With the Folders extension, all child tags have their parent tag in their name ('fruit.apples', 'fruit.bananas'). This takes up a lot of space in the note preview and tag fields and makes these areas hard to scan.

**Describe the solution you'd like**
Do not show parent tags in tag names on clients that have the Folders extension enabled.

**Comment by atmoio** [comment]
Native folders are planned for v4, so this will be resolved at that point. Currently folders are essentially a UI overlay on top of tags.

**Comment by myreli** [comment, marked as answer]
[Folders are now native](https://standardnotes.com/blog/folders-from-extension-to-native) and it seems like the issues here have been adressed. Let us know in a new issue if you enconter any problems. Thanks!
